File: Exam-Preps/SoftUniBabies/JavaScript/models/baby.js
const Sequelize = require('sequelize');

module.exports = function (sequelize) {
    let Baby = sequelize.define('Baby', {
        name: {
            type: Sequelize.TEXT,
            required: true,
            allowNull: false
        },

        gender: {
            type: Sequelize.TEXT,
            required: true,
            allowNull: false
        },

        birthday: {
            type: Sequelize.TEXT,
            required: true,
            allowNull: false
        }
    },{
        timestamps:false
    });

    return Baby;
};

File: Exam-Preps/SoftUniBabies/CSharp/SoftUniBabies/Models/Baby.cs
﻿namespace SoftUniBabies.Models
{
    using System;
    using System.ComponentModel.DataAnnotations;

    public class Baby
    {
        public int Id { get; set; }

        [Required]
        public string Name { get; set; }

        [Required]
        public string Gender { get; set; }

        [Required]
        public string Birthday { get; set; }
    }
}
